go/api/config.go

```diff
@@ -140,8 +140,7 @@ func (strategy *BackoffStrategy) toProtobuf() *protobuf.ConnectionRetryStrategy
 	}
 }
 
-// RedisClientConfiguration represents the configuration settings for a Standalone Redis client. baseClientConfiguration is an
-// embedded struct that contains shared settings for standalone and cluster clients.
+// RedisClientConfiguration represents the configuration settings for a Standalone Redis client.
```

Comment:
I realized that the final documentation does not show the unexported fields, so it doesn't make sense to mention baseClientConfiguration here